src/parameterized_tendencies/gravity_wave_drag/non_orographic_gravity_wave.jl

```diff
@@ -1,11 +1,17 @@
 #####
 ##### Non-orographic gravity wave parameterization
 #####
-
+using UnrolledUtilities
 import ClimaCore.Spaces as Spaces
 import ClimaCore.Fields as Fields
 import ClimaCore.Geometry as Geometry
-import ClimaCore.Operators as Operators
+import ClimaCore.Operators as Operator
+
+Operators.placeholder_space(current_space, parent_space) =
+    current_space isa Spaces.AbstractSpace &&
+    parent_space isa Spaces.AbstractSpace &&
+    Spaces.issubspace(current_space, parent_space) ?
+    Operators.PlaceholderSpace() : current_space
```

Comment:
This overrides a method in ClimaCore. If this is needed, it needs to be put into ClimaCore.

Reply:
Yes, it is needed. Otherwise the code could not run on GPU. Dennis tells me he will open a pull request in ClimaCore soon.